waku/waku_discv5.nim

```diff
@@ -52,7 +52,7 @@ type WakuDiscoveryV5* = ref object
     peerManager: Option[PeerManager]
     topicSubscriptionQueue: AsyncEventQueue[SubscriptionEvent]
 
-proc shardingPredicate*(record: Record): Option[WakuDiscv5Predicate] =
+proc shardingPredicate*(record: Record, bootnodes: seq[Record] = @[]): Option[WakuDiscv5Predicate] =
```

Comment:
This comment is out of the scope of the PR but, shall we make it private?
```suggestion
proc shardingPredicate(record: Record, bootnodes: seq[Record] = @[]): Option[WakuDiscv5Predicate] =
```

Reply:
it's required for testing